pkgs/os-specific/linux/kernel/manual-config.nix

```diff
@@ -301,10 +301,10 @@ let
         export HOME=${installkernel}
       '';
 
-      # Some image types need special install targets (e.g. uImage is installed with make uinstall)
+      # Some image types need special install targets (e.g. uImage is installed with make uinstall on arm)
       installTargets = [
         (kernelConf.installTarget or (
-          /**/ if kernelConf.target == "uImage" then "uinstall"
+          /**/ if kernelConf.target == "uImage" && stdenv.hostPlatform.isArmv7 then "uinstall"
```

Comment:
I think this should possibly just have been `isArm`.

Reply:
Good point! Follow up is here: https://github.com/NixOS/nixpkgs/pull/314622